XPlat/AzureMessaging/Android/samples/NotificationHubsSample/MainActivity.cs

```diff
@@ -1,192 +1,72 @@
-﻿using System;
-
-using Android.App;
-using Android.Content;
-using Android.Runtime;
-using Android.Views;
+﻿using Android.App;
 using Android.Widget;
 using Android.OS;
-using Android.Gms.Gcm;
-using Android.Gms.Iid;
-using WindowsAzure.Messaging;
-using Android.Gms.Iid;
-
-[assembly: Permission (Name = "@PACKAGE_NAME@.permission.C2D_MESSAGE")] //, ProtectionLevel = Android.Content.PM.Protection.Signature)]
-[assembly: UsesPermission (Name = "@PACKAGE_NAME@.permission.C2D_MESSAGE")]
-[assembly: UsesPermission (Name = "com.google.android.c2dm.permission.RECEIVE")]
-
-//GET_ACCOUNTS is only needed for android versions 4.0.3 and below
-[assembly: UsesPermission (Android.Manifest.Permission.GetAccounts)]
-[assembly: UsesPermission (Android.Manifest.Permission.Internet)]
-[assembly: UsesPermission (Android.Manifest.Permission.WakeLock)]
-[assembly: UsesPermission (Android.Manifest.Permission.ReceiveBootCompleted)]
+using WindowsAzure.Messaging.NotificationHubs;
+using Android.Content;
 
 namespace AzureMessagingSampleAndroid
 {
-    [Activity (Label = "Notification Hubs Sample", MainLauncher = true)]
+
+    [Activity(Label = "Notification Hubs Sample", MainLauncher = true)]
     public class MainActivity : Activity
     {
         TextView textViewMsg;
 
-        protected override void OnCreate (Bundle bundle)
+        protected override void OnCreate(Bundle bundle)
         {
-            base.OnCreate (bundle);
+            base.OnCreate(bundle);
 
             // Set our view from the "main" layout resource
-            SetContentView (Resource.Layout.Main);
+            SetContentView(Resource.Layout.Main);
+            textViewMsg = FindViewById<TextView>(Resource.Id.textViewMsg);
 
-            textViewMsg = FindViewById<TextView> (Resource.Id.textViewMsg);
+            // TODO: Customize these values to your own
+            const string CONNECTION_STRING = "YOUR-HUB-CONNECTION-STRING";
+            const string HUB_NAME = "YOUR-HUB-NAME";
 
-            #region Check to ensure user has used their own GCM Service, Hub Name, and Hub Secret
-            if (MyRegistrationService.GCM_SENDER_ID == "YOUR-SENDER-ID") {
-                Log ("You must change the SampleGcmBroadcastReceiver.SENDER_IDS to your own Sender ID for the sample to work!");
+            #region Check to ensure user has used their own Connection String and Hub Name
+            if (CONNECTION_STRING == "YOUR-HUB-CONNECTION-STRING")
+            {
```

Comment:
Nit on formatting to use braces on the same line

Reply:
it's c# :)